src/pages/ProductPage/ProdDetail/ProdDetail.scss

```
@@ -0,0 +1,28 @@
+.prodDetail{
+  font-weight: bold;
+
+  h1{

```

Let's avoid to use HTML tag directly, let's use BEM to mount our nested classes:

```suggestion
  &__title {
```